tools/typescript2fabel.js

```diff
@@ -266,6 +273,12 @@ function visitModule(node) {
             case ts.SyntaxKind.FunctionDeclaration:
                 mod.methods.push(getMethod(node, true));
                 break;
+            case ts.SyntaxKind.ModuleDeclaration:
+                modules.push(visitModule(node, modules));
+                break;
+            case ts.SyntaxKind.ClassDeclaration:
+                mod.interfaces.push(visitInterface(node));
+                break;
```

Comment:
Classes are a bit more complicated that interfaces because they can have constructors and static methods (and for that yo need a explicit reference to the class). We have to think yet how to translate properly. Luckily, they don't appear that often in definition files (I think).